main.cpp

```diff
@@ -221,18 +275,33 @@ struct Filter
     
     float cutOffFrequency = 20.0f;
 
-    int soundTone(float cutoffFrequency = 20.0f);
+    void getSoundTone(float cutoffFrequency = 20.0f);
     
     void addResonance(int rezAmount);
     
     void cutLowFreq(float cutOffFrequency = 1000.0f);
 };
 
-int Filter::soundTone(float cutoffFrequency)
+void Filter::getSoundTone(float cutoffFrequency)
 {
-    return {};
+    if (cutOffFrequency < frequencyRange)
+    {
+        float freqRangeRemaining;
+        freqRangeRemaining = (frequencyRange - cutOffFrequency);
+    }
+    else
+    {
+        std::cout << cutoffFrequency;
+    } 
+}
```

Comment:
Why don't you print out the freqRangeRemaining?  One of your blocks prints something, the other doesn't.
- [ ] revise.